src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
```

Comment:
This is border-color variable not border. I didn't see this value in the specs

Reply:
Updated to :
```
border-color: var(--sapList_BorderColor);
background: var(--sapList_Background);
```


The action item list has these values in spec defined
```
background: @sapUiButtonActionSelectBackground;
border-color: @sapUiButtonActionSelectBorderColor;
```

but it does not have CSS variable mapped.
<img width="860" alt="Screen Shot 2020-04-17 at 12 18 25 PM" src="https://user-images.githubusercontent.com/53487468/79540351-a5552000-80a5-11ea-801e-f8e64d3239ce.png">

When I check with Fiori3 team for the same they suggested using list relevant values for it.